af_data_adoxa_loader/models/calendar_appointment.py

```diff
@@ -151,26 +165,28 @@ def transform(self, row, transformations):
                 elif key == 'state':
                     if 'occasion_id' in row:
                         translation_dict = {
-                            'NULL':'free',
+                            'NULL':'confirmed',
                             '1':'done',
-                            '2':'free',
-                            '6':'confirmed',
-                            '7':'cancelled',
+                            '2':'done',
                         }
                     else:
                         translation_dict = {
-                            'NULL':'draft',
-                            '1':'booked',
-                            '2':'booked',
+                            'NULL':'request',
                             '6':'ok',
                             '7':'fail',
                         }
                     row[key] = translation_dict[row[key]]
-                elif key == 'location_id':
-                    row[key] = self.env['hr.location'].search([('location_code', '=', row[key])]).id
-                    keys_to_delete.append(key)
+                # elif key == 'location_id': #location is found through user_id.location_id
```

Comment:
Is there a reason for keeping this code commented out?

Reply:
Might be used in the future, added clarification to comment